src/sentry/reprocessing2.py

```diff
@@ -236,8 +229,15 @@ def reprocess_event(project_id: int, event_id: str, start_time: float) -> None:
             )
 
     if attachment_objects:
-        with sentry_sdk.start_span(op="reprocess_event.set_attachment_meta"):
-            attachment_cache.set(cache_key, attachments=attachment_objects, timeout=CACHE_TIMEOUT)
+        store_attachments_for_event(data, attachment_objects, timeout=CACHE_TIMEOUT)
+
+    # Step 2: Fix up the event payload for reprocessing and put it in event
+    # cache/event_processing_store
```

Comment:
Could switching Step 1 and Step 2 here become a problem? Not that I can think of any.